.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "00:00"
+      timezone: "America/Denver"
+    #assignees:
+    #  - "username"
+    #labels: need to add the required label to the repo first!
+    #  - "submodule-updates"
+    # must use the full team name, including the organization, as if you were @mentioning the team
+    #reviewers:
+    #  - "username"
```

Comment:
@coolbreeze413 Does it mean that it will create a PR without a default assignee and reviewer?

In general, it looks o.k. to me. @ganeshgore If you are fine, I propose to merge and see how it works after a few days.
@coolbreeze413 Are you fine to create any follow-up PR if we detect any issues?

Reply:
@tangxifan  yes the PR will not have any assignee or reviewer.
This can be set if you think that is better.

You can check an example PR here from my experiments: https://github.com/coolbreeze413/dependabot_main/pull/3

It can be merged by the bot by leaving a `@dependabot merge` comment as I tried here, or it can be manually merged.
Once the latest PR is merged, any previous unmerged PRs for the same submodule will be closed automatically as well, for example: https://github.com/coolbreeze413/dependabot_main/pull/2

Also, the bot will add a default `dependencies` label, if needed a custom one can be set as well.

I think it is a good idea to try it out, and then adjust the configuration as needed, I can raise PRs for the same accordingly, no problems.

Reply:
@tangxifan here is an example PR with the assignee and reviewer assigned automatically: https://github.com/coolbreeze413/dependabot_main/pull/4